### PR TITLE
if statement to redirect users activating accounts

### PR DESCRIPTION
**Pull request by Dinnall**
### What's this PR do?

This pull request send users to Northstar reg `page /about`, after they activate their account

### How should this be reviewed?
👁️ 
...

### Any background context you want to provide? 
We want to redirect a user to `profile/about` IF they've just activated their account.
...

### Relevant tickets

References [Pivotal #174675565](https://www.pivotaltracker.com/n/projects/2401401/stories/174675565).

### Checklist

- [ x ] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added appropriate feature/unit tests.
- [ ] If new attributes were added to users, there is a corresponding PR to surface these in Aurora.
- [ ] If new attributes were added to users, then the data team already knows about these changes.

### CODE REVIEW

File: app/Http/Controllers/Web/ResetPasswordController.php [hunk 1]
```diff
@@ -136,6 +136,10 @@ protected function guard()
      */
     public function redirectPath()
     {
+        if (Str::contains(request()->type, 'activate-account')) {
+            return 'profile/about';
+        }
+
```
Comment (katiecrane) [inline app/Http/Controllers/Web/ResetPasswordController.php:142]: Looks like StyleCI wants this to be a plain newline with no spaces, so if you remove the spaces from this line (but keep the blank line) it should be good!

Comment (Dinnall) [reply app/Http/Controllers/Web/ResetPasswordController.php]: I removed the space and moved the return to a new line. Currently still fail withing StyleCI

File: app/Http/Controllers/Web/ResetPasswordController.php [hunk 2]
```diff
@@ -136,6 +136,10 @@ protected function guard()
      */
     public function redirectPath()
     {
+        if (Str::contains(request()->type, 'activate-account')) 
+        {
+        return 'profile/about';
+        }
```
Comment (katiecrane) [inline app/Http/Controllers/Web/ResetPasswordController.php:142]: I think this is what it wants:
```suggestion
        if (Str::contains(request()->type, 'activate-account')) {
            return 'profile/about';
        }

```

**Review by katiecrane — APPROVED** [review]
looks good! I think to test on QA we'll have to trigger the activation via the API!

**Pull request merged**